app/_src/production/gui.md

```diff
@@ -27,8 +32,14 @@ When you run the GUI for the first time, you’ll be presented with the Wizard:
 ## Mesh Overview
 Once you’ve completed the setup process, you’ll be sent to the Mesh Overview. This is a general overview of all of the meshes found. You can then view each entity and see how many dataplanes and traffic permissions, routes, and logs are associated with that mesh.
 
-<center>
-<img src="/assets/images/docs/gui-mesh-overview.png" alt="A screenshot of the Mesh Overview of the Kuma GUI" style="width: 500px; padding-top: 20px; padding-bottom: 10px;"/>
+
+<center style="padding-top: 20px; padding-bottom: 10px;">
+{% if_version gte:2.6.x %}
+<img src="/assets/images/docs/2.6.x/gui/mesh-overview.png" alt="A screenshot of the Mesh Overview of the Kuma GUI" />
```

Comment:
```suggestion
<img src="/assets/images/docs/{{ page.version }}/gui/mesh-overview.png" alt="A screenshot of the Mesh Overview of the Kuma GUI" />
```

(will make this future proof :) )

Reply:
Ah perfect! Exactly what I was looking for! I'll do this locally with them all 👍

Reply:
Noticed that URLs with `/dev/` no longer work with this, no biggie right now but is there a way to make that work also do you know?